packages/portal-proto/src/features/facets/DateRangeFacet.tsx

```diff
@@ -71,7 +71,7 @@ const DateRangeFacet: React.FC<DateRangeFacetProps> = ({
       fromOp: ">=",
       toOp: "<=",
     };
-    console.log({ data });
+    // console.log({ data }); //TODO: confirm this should be removed
```

Comment:
I would just remove this, no reason to leave this as a TODO.

Reply:
I added the TODO and removed the commented-out console.log